systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -174,14 +175,14 @@ public void applyClusterOperatorInstallFiles(String namespace) {
                     (x, y) -> x,
                     LinkedHashMap::new));
         for (Map.Entry<File, String> entry : operatorFiles.entrySet()) {
-            LOGGER.info("Applying configuration file: {}", entry.getKey());
+            LOGGER.info("Creating configuration file: {}", entry.getKey());
             String fileContents = entry.getValue();
             if (Environment.isNamespaceRbacScope()) {
                 fileContents = switchClusterRolesToRoles(fileContents);
 
             }
             clusterOperatorConfigs.push(entry.getKey().getPath());
-            cmdKubeClient().namespace(namespace).applyContent(fileContents);
+            cmdKubeClient().namespace(namespace).replaceContent(fileContents);
```

Comment:
Is this going to need a change to the Strimzu upgrade documentation?

Reply:
This should be already done. The issue is that the Kafka CRD with all 3 schemas is too big for `kubectl apply`. I already went through the docs to change it to `kubectl create` for initial deployments and `kubect replace` for upgrade.